mbf_utility/include/mbf_utility/navigation_utility.h

```diff
@@ -107,6 +109,23 @@ bool getRobotPose(const TF &tf,
                   const std::string &global_frame,
                   const ros::Duration &timeout,
                   geometry_msgs::PoseStamped &robot_pose);
+/////////////////////////
+
+
+/**
+ * @brief subscribe odom_topic callback
+ * @param sub_odometry subscribe data from wheel odometry
+ */           
+void actual_vel_callback(const nav_msgs::Odometry::ConstPtr& sub_odometry);
```

Comment:
Please respect coding sttile: `actualVelCallback`

Reply:
I will change it.